tcms/core/management/commands/upgrade.py

```diff
@@ -0,0 +1,86 @@
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = "Perform post-upgrade tasks for Kiwi TCMS"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--noinput",
+            "--no-input",
+            "--yes",
+            "-y",
```

Comment:
It doesn't make sense to have so many options for the same thing. Choose one and go with it.